conda_smithy/cli.py

```diff
@@ -257,6 +260,22 @@ def __init__(self, parser):
             action="append",
             help="drone server URL to register this repo. multiple values allowed",
         )
+        scp.add_argument(
+            "--cirun-resources",
+            action="append",
+            help="cirun resources to enable for this repo. multiple values allowed",
+        )
+        scp.add_argument(
+            "--cirun-policy-args",
+            action="append",
```

Comment:
Same comment about `--cirun-resources` and `None` as default. We need some extra logic or config here to ensure it's properly handled.

Reply:
this property can be None